src/containers/onboarding/SkillsPrompt.js

```diff
@@ -1,35 +1,42 @@
 import React from 'react'
+import { updateUserSettings } from '../../actions'
 import Modal from '../../components/Modal'
 import ModalOnlyPage from '../../components/ModalOnlyPage'
 import ListItemTagInput from '../../components/ListItemTagInput'
-import { CommunityHeader } from '../Signup'
+import CommunityHeader from '../../components/CommunityHeader'
 import { nextOnboardingUrl } from '../../util/navigation'
-import { updateUserSettings } from '../../actions'
 import A from '../../components/A'
 import { find, map } from 'lodash/fp'
 const { func, object } = React.PropTypes
 
 const SkillsPrompt = ({ location, community }, { currentUser, dispatch }) => {
   community = community || find(c => c.slug === location.query.community,
     map('community', currentUser.memberships))
-  const update = (path, value) => dispatch(updateUserSettings({[path]: value}))
 
   const title = `Are there any skills, passions or interests you'd like to be
   known for in your community?`
 
   const subtitle = `Pick "tags" to describe yourself and to find people and
   opportunities that match your interests.`
 
+  const update = (path, value) => dispatch(updateUserSettings({[path]: value}))
+
+  const nextUrl = () => nextOnboardingUrl(location)
```

Comment:
i don't think this refactoring works as desired -- the property on the A tag below is getting set to this function, not the function's value